ci/fireci/fireciplugins/macrobenchmark.py

```diff
@@ -104,6 +104,19 @@ async def _copy_google_services():
     shutil.copyfile(src, dst)
 
 
+async def _post_processing(results):
+  # Upload successful measurements to the metric service
+  measurements = [i for x in results if not isinstance(x, Exception) for i in x]
```

Comment:
This line is getting a bit complex for a list comprehension, but feel free to leave it if you think is OK.

Reply:
Done. Use explicit for loop to assemble the list now.